R/pxweb_split_query.R

```diff
@@ -130,7 +132,7 @@ split_dimensions_left_right <- function(x, bool, max_size){
     prod_value <- batch_size/max_size
     if(prod_value > 1){
       if(i == 1) {
-        stop("\nToo large query. \nVariable(s) '", paste(names(x[!bool]), collapse = "', '"), "' cannot be split into batches (eliminate is set to FALSE by the API). \nThe smallest batch size is ", batch_size," and the maximum number of values that can be downloaded through the API is ", mxv, ". \nFor details and workarounds, see:\nhttps://github.com/rOpenGov/pxweb/blob/master/TROUBLESHOOTING.md", call. = FALSE)
+        stop("\nToo large query. \nVariable(s) '", paste(names(x[!bool]), collapse = "', '"), "' cannot be split into batches (eliminate is set to FALSE by the API). \nThe smallest batch size is ", batch_size," and the maximum number of values that can be downloaded through the API is ", .GlobalEnv$mxv, ". \nFor details and workarounds, see:\nhttps://github.com/rOpenGov/pxweb/blob/master/TROUBLESHOOTING.md", call. = FALSE)
```

Comment:
Same here. We should not assigne variables to the global environment.

Reply:
After some research adding the value of mxv to options seemed like the most straightforward way to pass it on to another function. Declaring it with globalVariables seemed a bit overkill for this purpose (funnily enough Hadley Wickham called globalVariables "a hideous hack" but [reversed his stance](https://stackoverflow.com/questions/9439256/how-can-i-handle-r-cmd-check-no-visible-binding-for-global-variable-notes-when#comment16961653_12429344) a few after that)

Reply:
Yes. But lets remove it so we get a NOTE, then I will fix this NOTE by adding the argument to the function and pass it before I submit it to CRAN.